ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6362,6 +6382,17 @@ VOADC2X:
   implements:
   - OPERATIONAL
 
+IDC:
```

Comment:
rename - this should indicate its percentage control

Reply:
Done